de/main.po

```diff
@@ -1,815 +1,881 @@
 msgstr ""
+"Du kannst die Sprache einer Nachricht erkennen, indem du darauf mit einem "
+"**globe** Emoji (%s) reagierst.\n"
+"Alternativ kannst du auch den Befehl `%s` verwenden:"
 
 msgid "Auto Delete Bot Commands"
-msgstr ""
+msgstr "Bot-Befehle automatisch löschen"
 
 msgid ""
 "You can choose to have the bot auto delete commands in the channel. All you "
 "have to do is write %s anywhere inside the channel's topic description."
 msgstr ""
+"Du kannst wählen, ob der Bot die Befehle in diesem Kanal automatisch löschen "
+"soll. Alles, was du tun musst, ist, %s irgendwo innerhalb der "
+"Themenbeschreibung des Kanals hinzuzufügen."
 
 msgid ""
 "This will delete all commands/messages including: help commands, settings, "
 "stats and translate commands for example:"
 msgstr ""
+"Dadurch werden alle Befehle/Nachrichten gelöscht, z.B.: Hilfebefehle, "
+"Einstellungen, Statistiken und Übersetzungsbefehle:"
 
 msgid "and"
-msgstr ""
+msgstr "und"
 
 msgid "Translate other Bots"
-msgstr ""
+msgstr "Andere Bots übersetzen"
 
 msgid ""
 "By default, Translator will ignore all messages by bots, this is a common "
 "practice to avoid unpredictable behaviours and response-loops."
 msgstr ""
+"Standardmäßig ignoriert Translator alle Nachrichten von Bots, dies ist eine "
+"gängige Praxis, um unvorhersehbare Verhaltensweisen und Antwortschleifen zu "
+"vermeiden."
 
 msgid ""
-"With that said, if you still wish to translate messages by other bots, you "
-"can do so by creating a member **role** called **can-translate** then "
-"assign it to the bots you wish to translate."
+"With that said, if you still wish to translate messages by other bots, you can "
+"do so by creating a member **role** called **can-translate** then assign it to "
+"the bots you wish to translate."
 msgstr ""
+"Wenn du dennoch Nachrichten von anderen Bots übersetzen möchtest, kannst du "
+"dies tun, indem du eine **role** mit dem Namen **can-translate** erstellst und "
+"diese dann den Bots zuweist, die du übersetzen möchtest."
 
 msgid "Disable/Stop"
-msgstr ""
+msgstr "Deaktivieren/Stoppen"
 
 msgid "Simply remove the %s role from the bots. Or delete the role itself."
-msgstr ""
+msgstr "Entferne einfach die %s-Rolle von den Bots. Oder lösche die Rolle selbst."
 
 msgid "Use at your own risk and report if you experience issues."
 msgstr ""
+"Benutze dies auf eigene Gefahr und melde dich, wenn du Probleme damit hast."
 
 msgid "Server commands are available only to server administrators."
-msgstr ""
+msgstr "Serverbefehle sind nur für Server-Administratoren verfügbar."
 
 msgid "Current Server Info"
-msgstr ""
+msgstr "Aktuelle Server-Informationen"
 
 msgid "Set servers's default language"
-msgstr ""
+msgstr "Die Standardsprache des Servers festlegen"
 
 msgid "Set servers's custom prefix"
-msgstr ""
+msgstr "Benutzerdefiniertes Präfix des Servers festlegen"
 
 msgid "Set servers's default style"
-msgstr ""
+msgstr "Festlegen des Standardstils des Servers"
```

Comment:
`Standardstil des Servers festlegen` sounds better and fits better the other translations.